app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -868,8 +868,10 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
 
             button_send.setOnClickListener {
                 var textMessage = citation ?: ""
```

Comment:
No need to have `var textMessage = citation ?: ""`
`if (!citation.isNullOrBlank()) { ... }` is the way to go.

Reply:
Thanks @filipedelimabrito for comments. I will update pr with necessary updates

Reply:
Let me know if I have to change the code. I will do it and update the pull request.